WalletWasabi.Gui/Program.cs

```diff
@@ -45,6 +46,9 @@ private static async Task Main(string[] args)
 				BuildAvaloniaApp()
 					.BeforeStarting(async builder =>
 					{
+						RuntimeParams.SetDataDir(Global.DataDir);
+						await RuntimeParams.LoadAsync();
```

Comment:
There must have been some storage that is loaded before the UI. RuntimeParams is a good option for that. 
Offtopic: If I store values in the UiConfig which are used in the Ui, I had some serious concurrency problems. UiConfig loaded parallel with the Ui elements and that is bad. It is working now, but try to add something to the UiConfig which used by the Statusbar...